tests/migration/Core/V6_6/Migration1738661307AddMediaIndicesTest.php

```diff
@@ -0,0 +1,79 @@
+<?php declare(strict_types=1);
+
+namespace Shopware\Tests\Migration\Core\V6_6;
+
+use Doctrine\DBAL\Connection;
+use PHPUnit\Framework\Attributes\CoversClass;
+use PHPUnit\Framework\TestCase;
+use Shopware\Core\Framework\DataAbstractionLayer\Dbal\EntityDefinitionQueryHelper;
+use Shopware\Core\Framework\Test\TestCaseBase\KernelTestBehaviour;
+use Shopware\Core\Migration\V6_6\Migration1738661307AddMediaIndices;
+
+/**
+ * @internal
+ */
+#[CoversClass(Migration1738661307AddMediaIndices::class)]
+class Migration1738661307AddMediaIndicesTest extends TestCase
+{
+    use KernelTestBehaviour;
+
+    private Connection $connection;
+
+    private string $tableName = 'media';
+
+    protected function setUp(): void
+    {
+        $this->connection = static::getContainer()->get(Connection::class);
+    }
+
+    public function testTimestamp(): void
+    {
+        static::assertSame(1738661307, (new Migration1738661307AddMediaIndices())->getCreationTimestamp());
+    }
+
+    public function testMigration(): void
+    {
+        $this->undoMigration();
+        // Test multiple execution
+        $this->migrate();
+        $this->migrate();
+
+        static::assertTrue($this->hasIndex('idx.media.file_extension', ['file_extension']));
+        static::assertTrue($this->hasIndex('idx.media.file_name', ['file_name']));
+        static::assertTrue($this->hasColumn('file_hash'));
+        static::assertTrue($this->hasIndex('idx.media.file_hash', ['file_hash']));
+    }
+
+    private function migrate(): void
+    {
+        (new Migration1738661307AddMediaIndices())->update($this->connection);
+    }
+
+    private function undoMigration(): void
+    {
+        if ($this->hasColumn('file_hash')) {
+            $this->connection->executeStatement(
+                <<<SQL
+                ALTER TABLE `{$this->tableName}` DROP COLUMN `file_hash`;
```

Comment:
I am always a bit allergic to such template strings in PHP, as they are often the entry point for SQL injections :grin: 

could you not just write "media" in all the places? having a class property with always the same static string is also not really needed :slightly_smiling_face: